README.md

```diff
@@ -47,40 +79,48 @@ Customize configurations are supported for `v1.0.2+`. There are several ways to
         }
     }
     ```
-3. By environment varialbe:
-    ```
+
+3. By environment variables:
+
+    ```shell
     AppInsightsForKubernetes__InitializationTimeout=3.1:12:15.34
     ```
 
     All the related configurations have to be put in a section named `AppInsightsForKubernetes`. The supported keys/values are listed below:
 
-    | Key                   | Value/Types | Default Value | Description                                                                                            |
-    |-----------------------|-------------|---------------|--------------------------------------------------------------------------------------------------------|
-    | InitializationTimeout | TimeSpan    | 00:02:00      |  Maximum time to wait for spinning up the container. Accepted format: [d.]hh:mm:ss[.fffffff]. |
+    | Key                   | Value/Types | Default Value | Description                                                                                  |
+    | --------------------- | ----------- | ------------- | -------------------------------------------------------------------------------------------- |
+    | InitializationTimeout | TimeSpan    | 00:02:00      | Maximum time to wait for spinning up the container. Accepted format: [d.]hh:mm:ss[.fffffff]. |
 
 The configuration uses with the conventions in ASP.NET Core. Refer [Configuration in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-2.1) for more information.

```

Comment:
The configuration uses ASP.NET Core conventions. Refer to...

Reply:
Yes